.github/workflows/ga_publish_ra_link.yml

```diff
@@ -0,0 +1,17 @@
+on: pull_request
+
+jobs:
+  example_comment_pr:
+    runs-on: ubuntu-latest
+    name: An example job to comment a PR
```

Comment:
This is from an example, can it be more meaningful for us

Reply:
removed